### ISSUE TITLE
[ENHANCEMENT] Get flutter and dart versions from branches

### ISSUE THREAD

**Issue by Faaatman** [kuhnroyal/flutter-fvm-config-action#7]
Github action breaks when `"flutterSdkVersion": "stable"` and is not a number.
Is there a way to get the flutter version from their stable branch?
**EDIT:**
### Temporary fix
```YAML
      - name: Flutter FVM config action
        uses: kuhnroyal/flutter-fvm-config-action@v1

      - name: Set up Flutter using FVM version number if specified
        if: ${{ env.FLUTTER_VERSION != 'stable' }}
        uses: subosito/flutter-action@v2
        with:
          flutter-version: ${{ env.FLUTTER_VERSION }}
          channel: ${{ env.FLUTTER_CHANNEL }}

      - name: Set up Flutter using FVM channel name
        if: ${{ env.FLUTTER_VERSION == 'stable' }}
        uses: subosito/flutter-action@v2
        with:
          channel: ${{ env.FLUTTER_CHANNEL }}
```
In my case, I'm just checking if it's `stable` since I won't be using other branches but you can add yours in the if statements.

**Comment by kuhnroyal** [comment]
I don't see this action solving this. Your workaround is probably the best way to go.
I also in general would suggest to use a fixed version on the stable channel like `2.8.1@stable`.
This allows the `subosito/flutter-action` to cache the version which results in faster builds because it does not need to check what the latest version on stable currently is.

**Comment by Faaatman** [comment]
>  also in general would suggest to use a fixed version on the stable channel like 2.8.1@stable.
This allows the subosito/flutter-action to cache the version which results in faster builds

Oh didn't know that!
Thanks I will do this!
